app.py

```diff
@@ -26,12 +36,14 @@ def render_template_strict(template_filename, **kwargs):
 
 
 def load_site_config() -> dict:
+    """Read main .yaml file containig high level website settings"""
     with open(SITE_CONFIG_FILEPATH, "r") as file:
         site = yaml.safe_load(file)
     return site
 
 
 def load_post_ymls() -> dict:
+    """Read and group in a dict all .yaml files containing a single post data"""
```

Comment:
It's unclear what the word "group" means here (it usually has a different meaning in CS context, i.e. https://www.w3schools.com/sql/sql_groupby.asp). Maybe something like "Read all the yaml files corresponding to posts, and store them in a dict whose key is the slug of each post and the value is the content of the yaml file."